charts/helmet/templates/_cronjob.yaml

```
@@ -0,0 +1,47 @@
+{{- define "helmet.cronjob" -}}
+{{- if .Values.cronjob.enabled -}}
+---
+apiVersion: {{ include "common.capabilities.cronjob.apiVersion" . }}
+kind: CronJob
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
```

Please use the namespace template:
```suggestion
  namespace: {{ include "common.names.namespace" . | quote }}
```